### PR TITLE
Updated names, labels, population of admin1/admin2

### PR DESCRIPTION
**Pull request by stepps00**
Fixes a portion of whosonfirst-data/whosonfirst-data#1640

This PR adds German, Dutch, English, and French label properties to macroregions, regions, and counties in Belgium. This PR also updates `name:` properties when needed, and updates population data for these records.

These values were verified through GeoNames, Statoids, Wikipedia, and Wikidata.

No PIP work required, can merge once approved.

### CODE REVIEW

File: data/102/049/907/102049907.geojson [hunk 1]
```diff
@@ -5,17 +5,38 @@
...
     "iso:country":"BE",
+    "label:deu_x_preferred_longname":[
+        "Bezirk Virton"
+    ],
+    "label:deu_x_preferred_placetype":[
+        "bezirk"
+    ],
     "label:eng_x_preferred_longname":[
-        "Virton Arrondissement"
+        "Arrondissement of Virton"
```
Comment (nvkelso) [inline data/102/049/907/102049907.geojson:20]: This is either grammatically wrong or is practically too long for the label.

Comment (nvkelso) [reply data/102/049/907/102049907.geojson:20]: The old value seems more appropriate. But prove me wrong!

Applies to all these records in the diff.

**Review by nvkelso — CHANGES_REQUESTED** [review]
English labels look wonky.

File: data/102/049/907/102049907.geojson (hunk 1 above)
Comment (stepps00) [reply data/102/049/907/102049907.geojson:20]: The "Arrondissement of *" format is coming from Wikidata and Wikipedia.. pages for this specific example below:

- https://www.wikidata.org/wiki/Q93860
- https://en.wikipedia.org/wiki/Arrondissement_of_Virton

I'm open to changing these values, but I wasn't able to verify on any source (wiki, geonames, government websites, or even social media / travel sites) that the "name + Arrondissement" format was used for these place names.. so I went with this format (storing the existing as a variant).

I can easily swap these values around in a new commit, if needed.

Comment (nvkelso) [reply data/102/049/907/102049907.geojson:20]: Hmm, okay.

I'm okay going with your suggestion but monitor customer feedback.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**